.travis.yml

```diff
@@ -4,7 +4,10 @@ addons:
     apt:
         packages:
             - libsuitesparse-dev
+env:
+  - PRE_INSTALL='setuptools>=18 scipy cython'
```

Comment:
I think you can remove *this* line, I wouldn't worry too much about it (if it works with no cython...).